inc/saibuffer.h

```diff
@@ -484,21 +484,29 @@ typedef enum _sai_buffer_profile_attr_t
      * @brief Dynamic threshold for the shared usage
      *
      * The threshold is set to the 2^n of available buffer of the pool.
-     * Mandatory when SAI_BUFFER_POOL_THRESHOLD_MODE = SAI_BUFFER_THRESHOLD_MODE_DYNAMIC
```

Comment:
ideally, i am looking for 

(SAI_BUFFER_PROFILE_ATTR_THRESHOLD_MODE == SAI_BUFFER_PROFILE_THRESHOLD_MODE_INHERIT_BUFFER_POOL_MODE &&
SAI_BUFFER_PROFILE_ATTR_POOL_ID.SAI_BUFFER_POOL_ATTR_THRESHOLD_MODE == SAI_BUFFER_POOL_THRESHOLD_MODE_STATIC) ||
(SAI_BUFFER_PROFILE_ATTR_THRESHOLD_MODE  == SAI_BUFFER_PROFILE_THRESHOLD_MODE_STATIC)

It my understanding that the first part is not supported, so itai make them in the comment and only added the second part in the metadata.